### PR TITLE
Add sql query command

### PR DESCRIPTION
**Pull request by sunny19930321**
This pr is mainly to add SQL query commands, while compatible with previous PQL command.
`bin/pinot-admin.sh PostQuery -brokerHost localhost -brokerPort 8000 -queryType sql -query "select * from baseballStats limit 10" `

`bin/pinot-admin.sh PostQuery -brokerHost localhost -brokerPort 8000 -queryType pql -query "select * from baseballStats limit 10" `

`bin/pinot-admin.sh PostQuery -brokerHost localhost -brokerPort 8000 -query "select * from baseballStats limit 10" `

### CODE REVIEW

File: pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/PostQueryCommand.java [hunk 1]
```diff
@@ -77,6 +81,10 @@ public PostQueryCommand setBrokerPort(String port) {
     _brokerPort = port;
     return this;
   }
+  public PostQueryCommand setQueryType(String queryType) {
```
Comment (fx19880617) [inline pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/PostQueryCommand.java]: add a line above.

File: pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/PostQueryCommand.java [hunk 2]
```diff
@@ -37,6 +38,9 @@
   @Option(name = "-brokerPort", required = false, metaVar = "<int>", usage = "http port for broker.")
   private String _brokerPort = Integer.toString(CommonConstants.Helix.DEFAULT_BROKER_QUERY_PORT);
 
+  @Option(name = "-queryType", required = false, metaVar = "<string>", usage = "Query use sql or pql.")
+  private String _queryType = "pql";
```
Comment (fx19880617) [inline pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/PostQueryCommand.java]: use Request.SQL

**Comment by fx19880617** [comment]
overall lgtm, some minor comments

File: pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/PostQueryCommand.java (hunk 1 above)
Comment (Jackie-Jiang) [reply pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/PostQueryCommand.java]: +1

Comment (sunny19930321) [reply pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/PostQueryCommand.java]: added. cc @fx19880617

File: pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/PostQueryCommand.java (hunk 2 above)
Comment (sunny19930321) [reply pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/PostQueryCommand.java]: cc.

**Pull request merged**